commands/apps.go

```diff
@@ -331,59 +373,48 @@ func RunAppsCreateDeployment(c *CmdConfig) error {
 	return c.Display(displayers.Deployments{deployment})
 }
 
-// waitForAppDeploymentRunning waits for a app deployment to be running.
-func waitForAppDeploymentRunning(apps do.AppsService, appID string, deploymentID string) (*godo.Deployment, error) {
-	failCount := 0
+func waitForActiveDeployment(apps do.AppsService, appID string, deploymentID string) error {
+	const maxAttempts = 1800
```

Comment:
In the version using a ticker, 1800 was the max number of seconds (it breaks on `i * interval > timeout`).  Moving to a loop, this is now the number of times that we go through the loop.  With the 10 second sleep at the end of the loop, `maxAttempts` should be `1800 / 10` for a half hour timeout.

Reply:
Oh! I get it. Thanks for explaining.